lib/collection.js

```diff
@@ -3237,6 +3237,9 @@ var writeConcern = function(target, db, col, options) {
     target.writeConcern = db.writeConcern;
   }
 
+  // NOTE: there is probably a much better place for this
```

Comment:
Is this something we'd actually want to change now?

Reply:
I actually initially thought this might be a bad idea, but it turns out that retryable writes only apply to commands that support write concerns anyway - so this might in fact be the correct place for it. I just don't like that it's sort of hidden inside.

Reply:
Either way, if we want to change this I think its fine during a subsequent point release